sdk/cosmos/azure-cosmos/azure/cosmos/aio/scripts.py

```diff
@@ -189,35 +196,38 @@ async def delete_stored_procedure(self, sproc, **kwargs):
     @distributed_trace_async
     async def execute_stored_procedure(
         self,
-        sproc,  # type: Union[str, Dict[str, Any]]
-        partition_key=None,  # type: Optional[str]
-        params=None,  # type: Optional[List[Any]]
-        enable_script_logging=None,  # type: Optional[bool]
-        **kwargs  # type: Any
-    ):
-        # type: (...) -> Any
+        sproc: Union[str, Dict[str, Any]],
+        partition_key: Optional[Union[str, int, float, bool]] = None,
+        params: Optional[List[Dict[str, Any]]] = None,
```

Comment:
like with the parameters in the query method, I opted into leaving these two variables in since they play an important part in the method's functionality